spring-expression/src/test/java/org/springframework/expression/spel/ExpressionLanguageScenarioTests.java

```diff
@@ -96,8 +96,7 @@ public void testScenario_DefiningVariablesThatWillBeAccessibleInExpressions() th
 		// Use the standard evaluation context
 		StandardEvaluationContext ctx = new StandardEvaluationContext();
 		ctx.setVariable("favouriteColour","blue");
-		List<Integer> primes = new ArrayList<>();
-		primes.addAll(Arrays.asList(2,3,5,7,11,13,17));
+		List<Integer> primes = new ArrayList<>(Arrays.asList(2, 3, 5, 7, 11, 13, 17));
```

Comment:
No need to create two lists in this context.

```suggestion
		List<Integer> primes = Arrays.asList(2, 3, 5, 7, 11, 13, 17);
```